src/ODriveUART.cpp

```diff
@@ -68,20 +78,22 @@ ODriveFeedback ODriveUART::getFeedback() {
 }
 
 String ODriveUART::getParameterAsString(const String& path) {
-    serial_ << F("r ") << path << F("\n");
+    serial_ << "r " << path << "\n";
     return readLine();
 }
 
 void ODriveUART::setParameter(const String& path, const String& value) {
-    serial_ << F("w ") << path << F(" ") << value << F("\n");
+    serial_ << "w " << path << " " << value << "\n";
 }
 
 void ODriveUART::setState(ODriveAxisState requested_state) {
-    setParameter(F("axis0.requested_state"), String((long)requested_state));
+    if(kMotorNumber == 0) setParameter("axis0.requested_state", String((long)requested_state));
+    if(kMotorNumber == 1) setParameter("axis1.requested_state", String((long)requested_state));
```

Comment:
this pattern does not scale well when adding more functions that use `setParameter("axis0...")`.